api/spec/v1.yaml

```diff
@@ -1475,6 +1496,31 @@ components:
       description: |
         A runtime transaction.
 
+    RuntimeTokenList:
+      type: object
+      properties:
+        tokens:
+          type: array
+          items:
+            $ref: '#/components/schemas/RuntimeToken'
+      description: |
+        A list of tokens on a runtime.
+
+    RuntimeToken:
+      type: object
+      properties:
+        token_addr:
```

Comment:
We should also add a field that exposes the evm (hex) address of the token's contract. We have the data already. Some name ideas: `evm_token_addr` or `token_evm_addr`, or `native` instead of `evm`, or similar.

And in `token_addr` description, we should then add something like `This address can be deterministically derived from token_evm_addr.`

Reply:
tracking in https://app.clickup.com/t/3u9y3yp